routers/users.py

```
@@ -0,0 +1,257 @@
+    if not user_record:
+        raise HTTPException(status_code=401,detail="없는 유저입니다.")
+
+    if not user_record or user_record["password"] != login_data.password:
+        print(f"DB비번: {user_record['password']}, 입력비번: {login_data.password}")
+        raise HTTPException(status_code=401, detail="인증 정보가 올바르지 않습니다.")
+
+    token, expires_in = create_access_token(user_record["user_id"])
+
+    return UserResponse(
+        access_token=token,
+        token_type="bearer",
```

Ruff (0.14.11)

Possible hardcoded password assigned to argument: "token_type"

(S106)